_fixtures/testunsafepointers.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2022 The Cockroach Authors.
```

Comment:
Does this need to be here?

Reply:
no, sorry; this was just my editor's template